tools/create-test-data.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import python_freeipa
+from python_freeipa import Client
+import random
+from faker import Faker
+fake = Faker()
```

Comment:
I'm surprised Black didn't insert two lines between the last imports and the first code line... Did you run Black on this file?